src/test/routes/logout.ts

```diff
@@ -21,11 +21,31 @@ describe('Logout receiver', () => {
   describe('on GET', () => {
     it('should remove session cookie', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      idamServiceMock.resolveInvalidateSession('ABC')
 
       await request(app)
         .get(AppPaths.logoutReceiver.uri)
         .set('Cookie', `${cookieName}=ABC`)
         .expect(res => expect(res).to.have.cookie(cookieName, ''))
     })
+
+    it('should remove session cookie even when session invalidation is failed ', async () => {
+      idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
```

Comment:
That is something that worries me. It is not being used at all for logout endpoint so remove it.

Please also add `attachDefaultHooks(app)` as in other route tests, having this would detect unused mock.

Reply:
done